tjmonopix2/analysis/plotting.py

```diff
@@ -479,9 +489,16 @@ def create_cluster_tot_plot(self):
             else:
                 plot_range = range(0, np.max(np.nonzero(self.HistClusterTot)))
 
+            tot_calib_file = self.configuration['scan'].get('tot_calib_file', None)
+            if tot_calib_file is not None:
+                x_axis_title = 'Cluster Charge [e⁻]'
```

Comment:
Personal preference, do we keep capitalization for every word? I would not by intuition

Reply:
I prefer it this way but will adapt to the used standard.